CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### Fixes
 
 - Improved legibility of python stack traces ([#434](https://github.com/databricks/dbt-databricks/pull/434)).
+- Streamlined sql connector output in dbt.log ([#437](https://github.com/databricks/dbt-databricks/pull/437))
```

Comment:
Can we please please please fix your auto formatter settings so we don't get so many changed lines in your pull requests 🙏 ?

Reply:
Better?